VacancyRssFeedService/WebRole.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.WindowsAzure;
+using Microsoft.WindowsAzure.Diagnostics;
+using Microsoft.WindowsAzure.ServiceRuntime;
+
+namespace VacancyRssFeedService
+{
+    public class WebRole : RoleEntryPoint
+    {
+        public override bool OnStart()
+        {
+            // For information on handling configuration changes
+            // see the MSDN topic at https://go.microsoft.com/fwlink/?LinkId=166357.
+
+            return base.OnStart();
```

Comment:
This should die if/when we switch to an App service

Reply:
I'll remove it